docs/authenticate.js

```diff
@@ -276,6 +288,65 @@ function initializeFirebase() {
   firebase.initializeApp(getFirebaseConfig(inProduction()));
 }
 
+const allReadBinding = Object.freeze({
+  role: 'roles/viewer',
+  members: ['allUsers'],
+});
+
+/**
+ * Lists assets in the current disaster's folder and for any that match a score
+ * asset path (either standard or backup) and are not already world-readable,
+ * send a request to make that asset world-readable. This should only be called
+ * when logged in as the GD user. It does not wait for these calls to complete,
+ * and does not print any errors, since it is just trying to help other users,
+ * and is not triggered by an explicit user action.
+ *
+ * Optimized for the common case of assets already being world-readable: the
+ * getIamPolicy is technically not necessary, but usually it will avoid a set,
+ * which should be at least as expensive.
+ */
+function makeScoreAssetsWorldReadable() {
+  // TODO(janakr): Switch to using listEeAssets once #368 is in.
+  // TODO(janakr): Consider sharing cache with list_ee_assets.js. Not trivial
+  //  because that code does additional EE requests to look at geometries, so
+  //  we would need a two-level cache, one raw and one with geometries.
+  ee.data.listAssets(eeLegacyPathPrefix + getDisaster(), {}, () => {})
+      .then((listResult) => {
+        if (!listResult) {
+          return;
+        }
+        const paths = new Set([getScoreAssetPath(), getBackupScoreAssetPath()]);
+        let foundAssets = 0;
+        for (const {id} of listResult.assets) {
+          if (paths.has(id)) {
+            foundAssets++;
+            ee.data.getIamPolicy(eeLegacyPrefix + id, () => {})
+                .then((policy) => {
+                  for (const binding of policy.bindings) {
+                    if (binding.role === 'roles/viewer') {
```

Comment:
maybe this is documented somewhere, what is this check for?

Reply:
Sorry, most of this was from trial and error/poking around in objects that I got back in the console. But added links in jsdoc above.